networkx/algorithms/connectivity/edge_augmentation.py

```diff
@@ -1124,15 +1124,16 @@ def complement_edges(G):
     >>> sorted(complement_edges(G))
     []
     """
+    adj = G.adj
```

Comment:
For now, it's faster to use `G._adj` per @dschult 's comment in #3700

Reply:
Done! Wasn't sure if that was official policy already or an ongoing discussion.